### ISSUE TITLE
Improve debug message more clear

### ISSUE THREAD

**Issue by ibocon** [dotnet/corefx#24641]
string _EventSource_VarArgsParameterMismatch_ in Microsoft.Diagnostics.tracing.EventSource.Redist could be improved showing lists of mismatch type and how they are not match.

> parameters to the Event method do not match the parameters to the WriteEvent method. This may cause the event to be displayed incorrectly.

**Comment by danmosemsft** [comment]
@vancem would you accept a PR for such a change...?

**Comment by vancem** [comment]
> @vancem would you accept a PR for such a change...?

Yes I would accept this change.    We actually have this information it is just a matter of localizing it so we can spit it out, and it is useful.   

However it is not a high priority because typically you can figure it out pretty easily.  (after all most events have < 3 parameters).   If someone in the community wants to do it, that would be great.  

+ @brianrob

**Comment by brianrob** [comment]
Also, this change should be made in the CoreCLR repo since that's where it will sync to the rest of the runtimes.  Microsoft.Diagnostics.Tracing.EventSource.Redist is currently just a static copy of those two - I'm working on getting it to sync as well.

**Comment by danmosemsft** [comment]
@brianrob you can talk to @safern if you want to reuse the git-syncing-daemon that works between corert and coreclr.

**Comment by brianrob** [comment]
@danmosemsft, yes!  I will reach out to @safern.  Thank you.
